meta-balena-reference/layers/meta-balena-bsp/conf/samples/bblayers.conf.sample

```diff
@@ -0,0 +1,19 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+POKY_BBLAYERS_CONF_VERSION = "2"
+
+BBPATH = "${TOPDIR}"
+BBFILES ?= ""
+
+BBLAYERS ?= " \
+    ${TOPDIR}/../layers/poky/meta \
+    ${TOPDIR}/../layers/poky/meta-poky \
+    ${TOPDIR}/../layers/meta-openembedded/meta-oe \
+    ${TOPDIR}/../layers/meta-openembedded/meta-filesystems \
+    ${TOPDIR}/../layers/meta-openembedded/meta-networking \
+    ${TOPDIR}/../layers/meta-openembedded/meta-python \
+    ${TOPDIR}/../layers/meta-balena/meta-resin-common \
+    ${TOPDIR}/../layers/meta-balena/meta-resin-sumo \
+    ${TOPDIR}/../layers/meta-balena-qemu \
```

Comment:
IIRC the changes in meta-balena-qemu were limited to grub and image customization. Would you consider getting rid of meta-balena-qemu and integrating those on meta-balena-common? I still think the goal is to get rid of integration layers completely and rely on hostapp device image builds. Qemu being the base images for all other device types could just be an integral part of meta-balena.